packages/e2e-test-utils-playwright/src/click-block-toolbar-button.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Clicks a block toolbar button.
+ *
+ * @this {import('./').TestUtils}
+ * @param {string} label  The text string of the button label.
+ * @param {string} [type] The type of button label: 'ariaLabel' or 'content'.
+ */
+export async function clickBlockToolbarButton( label, type = 'ariaLabel' ) {
```

Comment:
We don't need the `type` parameter if we rewrite the selector into a role selector.